### DISCUSSION TITLE
Help implementing into site for a complete newbie?

### DISCUSSION THREAD

**Discussion by sunnidelays** [AR-js-org/AR.js#354] [Help] [answered]
Hey guys, I'm trying to implement this into my brand website and if I'm being blunt, I have absolutely no idea what I'm doing. I'm very, very new to coding and having a lot of trouble understanding the tutorial. Is there anyone here that can help me with a more detailed walkthrough and explanation of the code?

**Comment by giaaaacomo** [comment, marked as answer]
hi! do you have basic html knowledge (did you build your own website or someone did that for you)? what type of ar are you interested in (marker-based, location-based or image-tracking-based)?

**Reply by sunnidelays** [reply]
I’m very new to HTML and still learning, so my website is made through Squarespace. I’m interested in image-tracking-based.

**Reply by giaaaacomo** [reply]
what kind of implementation are you looking for? do you want to have a full-page ar.js tab beside standard html tabs, do you want to create something like a window in a regular html page, or something different?

**Reply by sunnidelays** [reply]
Preferably an unlinked full-page tab. I plan to use an NFC chip to link to the tab, so only those with access to the chip can view the image tracker. I'm specifically looking to track an image on a garment, such as a hoodie, and play an animation over it. 

A window would also be nice to implement, as I can surround it with other elements.

**Reply by giaaaacomo** [reply]
first of all you could give a try to mindAR, which is specialized in image tracking and has a well done doc. for the NFC thing i don't have any kind of knowledge but that's totally something external to the AR lib, and should not be a problem. what you need to understand now is exactly how do you want to display the AR and find out what lib gives you the best results

**Reply by sunnidelays** [reply]
Okay, I'll give it a look! And the NFC chip idea is solely to house the link to the AR tracking page, so that's already handled. Thanks for the help!
